app/views/root/_base_page.html.erb

```diff
@@ -21,5 +21,7 @@
 </main>
 
 <% if local_assigns.fetch(:show_related_items, true) %>
-  <div id="related-items"></div>
+  <aside>
+    <div id="related-items"></div>
```

Comment:
So the template in static doesn't actually have a `#related-items` element. The `BodyInserter` processor in slimmer runs before the `RelatedItemsInserter`. The  `BodyInserter` moves the whole of the `wrapper` over to the template and the `#related-items` div is inside the wrapper so will be in the page_template by the time the `RelatedItemsInserter` runs. The `RelatedItemsInserter` `filter` method in the could be re-written as:

```ruby
def filter(content_document, page_template)
    if page_template.at_css('body.mainstream div#related-items')
      page_template.at_css('body.mainstream div#related-items').replace(related_item_block)
    end
  end
```

and it would have the same effect.

Thinking about it though I should probably just update the [`related_item_block`](https://github.com/alphagov/static/blob/master/app/views/root/related.raw.html.erb) template to use an aside and then this change will be rolled out everywhere we use related sidebars.